compress.js

```diff
@@ -3,10 +3,132 @@
 const infile = process.argv[2];
 const outfile = process.argv[3];
 
-const LZString = require('./lz_string.js');
-const fs = require('fs');
+const LZString = require("./lz_string.js");
+const fs = require("fs");
 
-fs.readFile(infile, {encoding: 'utf-8'}, (err,data) => {
-    const comp = LZString.compressToUTF16(data);
-    fs.writeFile(outfile,comp,{encoding:'utf-8'}, (err) => {});
+const normalize_data = (names_list) => {
```

Comment:
This file is changed completely. It still uses LZString but the base64Encoding version, combined with a different data representation.